src/SlmQueue/Job/JobInterface.php

```diff
@@ -33,4 +33,13 @@ public function getId();
      * @return void
      */
     public function execute();
+
+    /**
+     * Chain a job.
+     *
+     * When the job has finished these jobs will be added to the queue in the same order they where added.
+     *
+     * @param AbstractJob $job
+     */
+    public function chainJob(AbstractJob $job);
```

Comment:
I'm against the add in the interface, it should stay easy. Maybe add a new ChainableJobInterface?